homework/main.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+document.addEventListener('DOMContentLoaded',function(event){
+  var  ToDoContentManager = new ToDoContentManager();
+  var  ToDoListManager = new ToDoListManager();
+  ToDoContentManager.start();
```

Comment:
Please don't overwrite existing classes with the instances. You should rename them as `toDoContentManager` and `toDoListManager`.